### PR TITLE
Add primitive types

### PR DESCRIPTION
**Pull request by JulianPasquale**

### CODE REVIEW

File: src/slides/class1.jsx [hunk 1]
```diff
@@ -210,5 +212,210 @@ export default () => (
...
+
+          console.log(foo); // undefined
+          typeof foo // "undefined"
+        `}
+      </CodePane>
+    </Slide>
+
+    <Slide backgroundColor="#ffd84d">
+      <Heading margin="0px" fontSize="50px">
+        Null
```
Comment (nachosoloaga) [inline src/slides/class1.jsx:293]: Might be a good idea to add something about Null vs undefined? Maybe not here, but at the time speaking

File: src/slides/class1.jsx [hunk 2]
```diff
@@ -210,5 +212,210 @@ export default () => (
...
+                `}
+            </CodePane>
+
+            <iframe src="/src/slides/class1/examples/boolean4.html"></iframe>
+          </FlexBox>
+        </Grid>
+      </div>
+
+      <Notes>
+        No confundir el objeto Booelan con los valores primitivos Boolean ni con la función Boolean.
```
Comment (nachosoloaga) [inline src/slides/class1.jsx:415]: Typo in the first Boo**e**lan

Comment (nachosoloaga) [reply src/slides/class1.jsx:415]: Also we should see how we are going to explain this in order to not make it confusing

Comment (JulianPasquale) [reply src/slides/class1.jsx:415]: yeah, agree. This note is after all the examples, so the idea is to explain the reason of the "weird" behavior in the last one.
I guess we'll see if that make sense when all is complete 🤣

File: src/slides/class1.jsx [hunk 3]
```diff
@@ -417,5 +416,131 @@ export default () => (
...
+      </CodePane>
+    </Slide>
+
+    <Slide backgroundColor="#ffd84d">
+      <Heading margin="0px" fontSize="50px">
+        BigInt
+      </Heading>
+
+      <Text>
+        El tipo BigInt permite representar números con una presición arbitraria, es decir que permite operar con números que superen los límites planteados por el tipo Number.
```
Comment (nachosoloaga) [inline src/slides/class1.jsx:492]: `presición` por `precisión`

**Review by nachosoloaga — APPROVED** [review]

**Pull request merged**